lib/providers/services/firebase/firebase_remote_config.dart

```diff
@@ -27,7 +27,8 @@ class FirebaseRemoteConfigService {
   }
 
   Future _fetchAndActivate() async {
-    await _remoteConfig.fetchAndActivate();
+    // this method is not defined
```

Comment:
this method does nothing then ?

Reply:
I'm going to check in the doc lib

Reply:
Apparently it already works, I think it appeared as an error when I had not updated correctly my packages.